code/datums/components/holderloving.dm

```diff
@@ -0,0 +1,73 @@
+/** Holder Loving Component
+ *
+ * This component is assigned to an [/obj/item], and also keeps track of a [holder].
+ * The [parent] is 'bound' to [holder]. [parent] will be kept either directly
+ * inside [holder], or in the inventory of a [/mob] that is itself holding [holder].
+ *
+ * If [parent] is placed in a [loc] that is not [holder] or [holder].[loc]
+ * (if it's a mob), it is placed back inside [holder].
+ *
+ * This is intended for items that are a 'part' of another item.
+ *
+ * It can also delete [parent] when [holder] is deleted.
+ *
+ */
+/datum/component/holderloving
+	can_transfer = TRUE
+	/** Item that parent is bound to.
+	 * We try to keep parent either directly in holder, or in holder's loc if loc is a mob,
```

Comment:
```suggestion
	/**
	 * Item that parent is bound to.
	 * We try to keep parent either directly in holder, or in holder's loc if loc is a mob,
```

I believe this is required for doc comments.

Reply:
Looks better too, in either case